docs/docs/guides/testing.md

```diff
@@ -2,27 +2,27 @@
 title: Testing
 ---
 
-## Testing Recoil Selectors inside of a React Component
+## Testing Recoil state inside of a React component
```

Comment:
Just for consistency, `state` in the second block is capitalized (`State`). Shouldn't be this one capitalized too? Or the other one in lower case.

Reply:
thanks for the catch!